src/data/general-cs-quiz.ts

```diff
@@ -21,6 +21,17 @@ const generalCSQuiz = [
       "Quick sort works by selecting one element (called “the pivot”) and finding the index where the pivot should end up in the sorted array.",
     Link: "https://www.freecodecamp.org/news/introduction-to-algorithms-with-javascript-examples/#quick-sort",
   },
+  {
+    Question:
+      "Which of these sorting algorithms follows an average case time complexity of order O(nlogn)?",
+    Answer: "Merge Sort",
+    Distractor1: "Selection Sort",
+    Distractor2: "Bubble Sort",
+    Distractor3: "Insertion Sort",
+    Explanation:
+      "The average case time complexity of Merge Sort Algorithm is O(nlogn) whereas for the other 3 options it is O(n^2).",
```

Comment:
I don't think Merge Sort Algorithm needs to be capitalized. 

```suggestion
      "The average case time complexity for the merge sort algorithm is O(n log n).",
```